README.md

```diff
@@ -27,6 +27,76 @@ https://eui-event.vercel.app/
 - Run `yarn start dev` to start it up!
 - Go to localhost:3000 in your browser to view it!
 
+## Connect Google Sheets with your app
+
+Follow these steps to connect your google sheets file with the app.
+
+### 1. Create a google sheet
+
+#### 1. Go to https://docs.google.com/ and create a blank sheet
+#### 2. Add name, title, team, location, shortBio, pronouns, imageLink headers in the sheet.
+#### 3. Populate the sheet with some data.
+#### 4. Change the sheet name to "Speakers" (Name is case sensitive).
+
+![](https://i.postimg.cc/MGPDVgkF/Sheet-Demo.jpg)
+
+
+### 2. Deploy App Script Web App
+
+#### 1. Open your google sheet.
+#### 2. Click on Extensions tab.
+#### 3. Click on App Script.
+![](https://i.postimg.cc/x8BPkmzp/App-Script.jpg)
+#### 4. Delete all the code from the editor (inside Code.gs file).
+####
+        function doGet(req) {
+            if(req.parameters.sheetName == "Speakers") {
+                return getSpeakersData()
+            }
+        }
+
+        function getSpeakersData() {
+        var doc = SpreadsheetApp.getActiveSpreadsheet()
+        var sheet = doc.getSheetByName("Speakers")
+        var values = sheet.getDataRange().getValues()
+        
+        var output = []
+        for(var i=1; i<values.length; i++) {
+            var row = {}
+            row['name'] = values[i][0]
+            row['title'] = values[i][1]
+            row['team'] = values[i][2]
+            row['location'] = values[i][3]
+            row['shortBio'] = values[i][4]
+            row['pronouns'] = values[i][5]
+            row['imageLink'] = values[i][6]
+
+            output.push(row)
+        }
+
+            return ContentService.createTextOutput(JSON.stringify({speakers: output})).setMimeType(ContentService.MimeType.JSON)
+        }
+        
+#### 5. Paste above code in the Code.gs file.
+#### 6. Click on Deploy button and select New Deployment.
+![](https://i.postimg.cc/43DGfL8r/New-deployment.jpg)
+
+#### 7. Click on settings icon on select type menu and select web app.
+![](https://i.postimg.cc/jjW3jRmj/Deployment-settings.jpg)
+
+#### 8. Add Description.
+#### 9. Select Anyone in who has access section and click deploy.
+#### 10. Allow all permission if asked.
```

Comment:
Probably need to clarify at this point, click `Deploy`, THEN authorize access if asked.

Reply:
Also might want to add that Google might say it's an unverified app and tell you to go `back to safety` and do that whole thing, and maybe explain what it is and that this is okay because it's something you're developing and it's essentially you accessing your own data.

Reply:
Noted.